lib/tests.nix

```diff
@@ -30,7 +29,8 @@ let
   mkTestDerivationFromNixvimModule =
     {
       name ? null,
-      pkgs ? defaultPkgs,
+      pkgs ? null,
```

Comment:
Yes, for backwards compatibility. I've tried to make this a non-breaking change.

IMO rather than deprecating specific args, it makes more sense to deprecate all functions in `lib/tests.nix` and `wrappers/standalone.nix`. That way they can be replaced by functions that have no baggage, do not need to be per-system in the flake output, and can more closely wrap the underlying `evalNixvim`.

I'm not ready to do that in this PR, and it's already large enough as it is. Before deprecating the per-system functions, I think we should move the "standalone package" into the module configuration; `modules/top-level/output.nix`.